friends/src/App.js

```diff
@@ -20,10 +17,23 @@ class App extends Component {
     .catch(err => {console.log(err)})
   }
 
+  deleteFriend = (e, id) => {
```

Comment:
since you don't need to do anything with this event, you can neglect passing it in at all